pages/code-review.md

```diff
@@ -3,4 +3,125 @@ permalink: /code-review/
+  If they are, use them with aplomb.
+- Talk offline if there is too much back and forth. Post a follow-up comment
+  summarizing the discussion.
+- Praise team members when they create exemplary work or suggestions.
+
+### For code submitters
+
+- Link to the code review from the originating task/issue, if applicable.
+- Remember that the code isn't you, don't get defensive when a reviewer is critical
+  of the code.
+- Seek to understand the reviewer's perspective.
+- Try to respond to every comment.
+- Be grateful for the reviewer's suggestions. ("Good catch, fixing in a4994ec")
+- Explain why the code exists. ("We need to work around these existing patterns")
+- Extract out-of-scope changes and refactorings into future tasks/issues.
+- Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+
+### For code reviewers
+
+- Understand why the code is necessary (bug, user experience, refactoring)
+- Seek to understand the author's perspective.
+- Clearly communicate which ideas you feel strongly about and those you don't.
+- Identify ways to simplify the code while still solving the problem.
+- Offer alternative implementations, but assume the author already considered
+  them. ("What do you think about such-and-such here?")
+- Sign off on the pull request with a :thumbsup: or "Ready to merge" comment.
+- Wait to merge the branch until it has passed Continuous Integration testing.
+  (TDDium, TravisCI, etc.)
+
+### Who merges
+
+There's lively debate over whether the code author or reviewer should merge
+the pull request. Follow the idioms that your team has set; if none are
+present, discuss until you have a consensus. Write that down to resolve the
+issue quickly in the future.
+
+## <a name="submitting"></a>Submitting code
+
+Before seeking a review, you should be able to check off each of the following:
+
+    - [ ] Changes are limited to a single goal (no scope creep)
+    - [ ] Code can be automatically merged (no conflicts)
+    - [ ] Code follows the standards laid out in this playbook
+    - [ ] Passes all existing automated tests
+    - [ ] New functions include new tests
+    - [ ] New functions are documented (with a description, list of inputs, and
+         expected output)
+    - [ ] Placeholder code is flagged
+    - [ ] Visually tested in supported browsers and devices
+    - [ ] Project documentation has been updated (including the "Unreleased" section of
+          the CHANGELOG)
+
+### Opening a PR
+
+Once sure your code meets the checks above, open a Pull Request for your new
+feature or bugfix following CFPB's [PR Template](https://raw.githubusercontent.com/cfpb/front-end/master/pr-template.md). You can also create a [Bookmarklet based on this template](https://gist.github.com/cfarm/b9b638943a2eea52a3a8). Remember that the
```

Comment:
interesting! Haven't seen anyone use this template -- you think we should still include it?